WhisperAPI/WhisperAPI/Models/ConversationContext.cs

```diff
@@ -10,15 +11,30 @@ public ConversationContext(Guid chatkey, DateTime datetime)
         {
             this.ChatKey = chatkey;
             this.StartDate = datetime;
+            this.MessagesSuggestions = new List<MessageSuggestion>();
```

Comment:
When you see yourself duplicating code, it should ring a bell, it is usually bad practice (although sometimes it is necessary). In this case, you should chain your constructors. The specialized constructor (the one with chatkey and DateTime) should be calling the parameterless constructor at the beginning. This way, you only have to initialize your list once.